### DISCUSSION TITLE
Explore feed is too slow to have it as the default Explore page

### DISCUSSION THREAD

**Discussion by lucasyvas** [community/community#18140] [Mobile] [answered]
I actually don't mind the new Explore capabilities added in mobile recently - it's basically a news feed. Honestly, not my favorite, but I won't split hairs.

What I will say is that I was a heavy visitor of Trending Repositories, and now I'm forced to wait for the feed to load before I can go there. The feed is one of the slower loading app pages I've encountered, so it's a slight nuisance. If you could navigate before loading was complete, I think that would be a quick fix that was satisfactory.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note here, I agree this could be better! I've flagged your suggestion to the team :)

**Reply by lucasyvas** [reply]
Appreciate the exploration to make a little more... non-blocking! I love how the app is shaping up by the way - a great mobile companion.
